Makefile

```diff
@@ -33,7 +34,7 @@ db: python
 	@tox -qe dev  --run-command 'alembic -c conf/alembic.ini $(args)'
 
 .PHONY: dev
-dev: build/manifest.json python
+dev: node_modules/.uptodate python
```

Comment:
Something I'm not sure about here is whether this could introduce a race condition if the Python app starts up and is ready to go before the first frontend build completes. I would suggest to check what happens when `build/manifest.json` does not exist at the point when `make dev` is run and then `build/manifest.json` is not built for the first time until after the Python app has started.